README.md

```diff
@@ -388,51 +388,72 @@ For a developer guide, see the [translating dev doc](docs/translating-dev.md).
+   - __[T-Enhancement](https://github.com/vector-im/element-web/labels/T-Enhancement):__
+     New features, changes in functionality, performance boosts, user-facing
+     improvements
+   - __[T-Task](https://github.com/vector-im/element-web/labels/T-Task):__
+     Refactoring, enabling or disabling functionality, other engineering tasks
+   - __[T-Other](https://github.com/vector-im/element-web/labels/T-Other):__
+     Questions, user support, anything else
+
+2. __Severity__ — All issues labeled `T-Defect` are also assigned a severity:
+   * __[S-Critical](https://github.com/vector-im/element-web/labels/S-Critical):__
+     Prevents work, causes data loss, affects many users, and/or has no
+     workaround
+   * __[S-Major](https://github.com/vector-im/element-web/labels/S-Major):__
+     Severely degrades major functionality or product features, with no
+     satisfactory workaround
+   * __[S-Minor](https://github.com/vector-im/element-web/labels/S-Minor):__
+     Impairs non-critical functionality, or suitable workarounds exist
+   * __[S-Tolerable](https://github.com/vector-im/element-web/labels/S-Tolerable):__
+     Purely cosmetic or low / no impact to users
+
+3. __Priority__ — All issues which are not `T-Other` are assigned a priority:
+   * __[P1](https://github.com/vector-im/element-web/labels/P1):__ Next
+   * __[P2](https://github.com/vector-im/element-web/labels/P2):__ Later
+   * __[P3](https://github.com/vector-im/element-web/labels/P3):__ Eventually
+   * __[P4](https://github.com/vector-im/element-web/labels/P4):__ Interesting —
+     Not yet scheduled, will accept patches
+   * __[P5](https://github.com/vector-im/element-web/labels/P5):__ Dubious —
+     Will not schedule, would consider patches
+
+4. __Area__ — Most issues are assigned one or several "areas" using one of the
+   many `A-` prefixed labels, e.g. `A-Composer` or `A-Spaces`. Each area label
+   maps to a group of features or portion of the UI surface in the app.
+
+### Other common labels
+
+We have a handful of other labels which are added on an as-needed basis, and not expected to be exhaustive:
+
+* __Exceptions__ — Special flags for issues and pull requests:
+  * __[X-Needs-Info](https://github.com/vector-im/element-web/labels/X-Needs-Info):__
+    This issue is blocked pending further information from the reporter
+  * __[X-Regression](https://github.com/vector-im/element-web/labels/X-Regression):__
+    Denotes things breaking which previously worked
+  * __[X-Release-Blocker](https://github.com/vector-im/element-web/labels/X-Release-Blocker):__
+    Issues which must be resolved before making a release
+
+* __[Easy](https://github.com/vector-im/element-web/labels/Easy)__ / __[Help
+  Wanted](https://github.com/vector-im/element-web/labels/Help%20Wanted)__ —
+  Well-defined issues which are suitable for folks new to the codebase
+
+* __[Privacy](https://github.com/vector-im/element-web/labels/Privacy)__ /
```

Comment:
Perhaps also list the few other "unprefixed" labels like `A11y`, `I18n` as well.

Reply:
These have now been added.